model/database/cosmosdb-metrics.yaml

```diff
@@ -0,0 +1,48 @@
+groups:
+  - id: metric.db.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.cosmosdb.operation.request_charge
+    brief: "[Request charge](https://learn.microsoft.com/azure/cosmos-db/request-units) consumed by the operation"
+    instrument: histogram
+    unit: "{request_unit}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.client.active_instances
+    type: metric
+    metric_name: db.cosmosdb.client.active_instances
+    brief: "Number of active client instances"
+    instrument: histogram
```

Comment:
this one should probably be updowncounter - you probably don't expect people to measure P95 for it. Also it does not change much in well-behaved application.

We have plenty of examples of current/active metrics in semconv - `http.server.active_requests`,  `db.client.connection.count`, `dotnet.thread_pool.thread.count`, etc all of them are `updowncounters`.

Histogram will be more expensive but also harder to use and would require you to keep a current value around. With up-down-counter, you'd record +1 when client is created and -1 when it's disposed and won't need to maintain any state in the clients.

Reply:
I have changed to `updowncounter`, but it means, dimensions would be <html>

Tag/dimension name | Sample value
-- | --
db.system | cosmodb
db.collection.name | myCollectionName
db.namespace | myDatabaseName
server.address | myaccountname.documents.azure.com
server.port | 443